consultation_analyser/consultations/public_schema.py

```diff
@@ -1,203 +1,105 @@
-            "I would like clarification on a few key points",
-        ],
-    )
-
-
-class ConsultationResponse(BaseModel):
-    """
-    A ConsultationResponse groups answers. For now it is also a placeholder for response-level information such as demographics, responding-in-the-capacity-of, etc.
-    """
-
-    model_config = ConfigDict(
-        extra="forbid",
-    )
-    submitted_at: AwareDatetime = Field(
-        ..., description="The submission date and time of the response"
-    )
-    answers: List[Answer] = Field(..., description="The answers in this response", min_length=1)
-
-
-class Section(BaseModel):
-    """
-    A Section contains a group of Questions. Consultations that do not have multiple sections should group all Questions under a single Section.
-    """
-
-    model_config = ConfigDict(
-        extra="forbid",
-    )
-    name: str = Field(
-        ...,
-        description="The name of the section",
-        examples=[
-            "When to enforce a Kit Kat ban",
-            "When to encourage the eating of Kit Kats",
-            "When Kit Kats are consumed",
-        ],
-    )
-    questions: List[Question] = Field(
-        ..., description="The questions in the consultation", min_length=1
-    )
+    question_part: QuestionPart
+    respondent: Respondent
+    text: str
+    chosen_options: Dict[str, Any] = Field(..., description="JSON object data")
 
 
-class Consultation(BaseModel):
-    """
-    Consultation is the top-level object describing a consultation. It contains one or more Sections, which in turn contain Questions.
-    """
-
-    model_config = ConfigDict(
-        extra="forbid",
-    )
-    name: str = Field(
-        ...,
-        description="The name of the consultation",
-        examples=[
-            "Consultation on Kit Kats",
-            "How should Kit Kats change",
-            "What shall we do about Kit Kats",
-        ],
-    )
-    sections: List[Section] = Field(
-        ..., description="The sections of the consultation", min_length=1
-    )
-
-
-class ConsultationWithResponses(BaseModel):
-    """
-    A Consultation and its ConsultationResponses
-    """
-
-    model_config = ConfigDict(
-        extra="forbid",
-    )
-    consultation: Consultation = Field(..., description="The consultation")
-    consultation_responses: List[ConsultationResponse] = Field(
-        ..., description="The responses", min_length=1
-    )
-
-
-class ConsultationWithResponsesAndThemes(BaseModel):
-    """
-    A Consultation and its ConsultationResponses, plus Themes assigned by the tool
-    """
-
-    model_config = ConfigDict(
-        extra="forbid",
-    )
-    consultation: Consultation = Field(..., description="The consultation")
-    consultation_responses: List[ConsultationResponse] = Field(
-        ..., description="The responses", min_length=1
-    )
-    themes: List[Theme] = Field(..., description="The themes", min_length=1)
+class Framework(BaseModel):
```

Comment:
Potentially don't need all of these models as we either won't be reading in this data (ThemeMappings will be created on import?), or they come from Pydantic models in `themefinder`, but given the uncertainty, can leave them in for now.